packages/handler-kit-azure-func/README.md

```diff
@@ -0,0 +1,49 @@
+# @pagopa/handler-kit-azure-func
+
+`@pagopa/handler-kit` adapter for `Azure Functions`
+
+### How to use it
+
+```typescript
+import { httpAzureFunction } from "@pagopa/handler-kit-azure-func";
+
+// Given an Handler
+// (from the @pagopa/handler README example)
+
+const GetMovies = H.of((req: H.HttpRequest) =>
+  pipe(
+    req.body,
+    // perform a refinement with io-ts, and returns a ValidationError
+    // that represents a 422 HTTP response
+    H.parse(GetMoviesBody),
+    E.map(({ genre }) => genre),
+    RTE.fromEither,
+    RTE.chainTaskEither(getMoviesByGenre),
+    RTE.map((movies) => ({ items: movies })),
+    // wrap in a 200 HTTP response, with content-type JSON
+    RTE.map(H.successJson),
+    // convert Error instances to problem json (RFC 7808) objects
+    RTE.orElseW(flow(H.toProblemJson, H.problemJson))
```

Comment:
does this logs something? may we add the logger to this example in case it doesn't? (or add a comment explaining that it logs the error)

Reply:
in this case, this `logs` (with `debug` level) the content of the input (the `HttpRequest`) and not the error response (because the error response it's catched by the handler)

Reply:
this is not very clear. what if I don't want this behavior? maybe should be opt-int?

Reply:
I think it's necessary to always log the error like debugging the request.